src/main.rs

```diff
@@ -75,7 +77,18 @@ async fn main() {
             verbose: _,
         } => {
             let (tx, mut rx) = mpsc::channel::<Vec<String>>(400);
-            let list_repos = ListReposForOrg::new(&org);
+            let org_name = if let Some(org_name) = config.github.org {
+                org_name
+            } else {
+                org
+            };
+            let num_days: i64 = if let Some(number_of_days) = config.github.number_of_days {
+                number_of_days.try_into().unwrap()
+            } else {
+                30
```

Comment:
Oh cool! Should we make this a Clap arg? I included it in the `Config` instead of `Opt`, but I could see that maybe making more sense as a CLI arg rather than a config property. 

But agreed that this can take place in a separate PR (maybe the one that refactors the command-line args #26?)

Reply:
Yep. I think I'd like to merge the idea of "toml config" and command line options, but I'm not sure the best way to do that. I imagine being able to load options from either place, with the command line taking precedence.